.github/workflows/docker.yml

```diff
@@ -19,77 +19,225 @@ on:
   push:
     branches:
       - master
+      - main
+
+  release:
+    types:
+      - published
 
   pull_request:
 
-# Certain actions will only run when this is the master repo.
+# Certain actions will only run when this is the master/main repo.
 env:
-  MASTER_REPO: EDModel/ED2
-  DOCKERHUB_ORG: pecan
+  MAIN_REPO: EDmodel/ED2
+  DOCKERHUB_ORG: edmodel
```

Comment:
Nice catch, though we should probably acknowledge that this code is based on PEcAn somewhere.